ambari-logsearch/ambari-logsearch-logfeeder/src/main/java/org/apache/ambari/logfeeder/util/FileUtil.java

```diff
@@ -121,11 +121,11 @@ public static File getFileFromClasspath(String filename) {
             return files;
           }
         } catch (Exception e) {
-          LOG.warn("Input file not found by pattern (exception thrown); {}, message: {}", searchPath, e.getMessage());
+          LOG.warn("Input file has not found by pattern (exception thrown); {}, message: {}", searchPath, e.getMessage());
```

Comment:
Original wording is better (applies to both log statements).  It's not the "input file" that is trying to find something, is it?  So please keep the original one, or change to "was not found" or "has not been found".